install_config/upgrading/os_upgrades.adoc

```diff
@@ -0,0 +1,27 @@
+[[install-config-upgrading-os-upgrades]]
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+When you need to upgrade the Operating System, either changing OS
+versions or just updates to the system software, you need to be aware
+that this can have an impact on the {product-title} software running
+on those machines.
+
+In particular, these updates can affect the `iptables` rules that
```

Comment:
We update the rules *if you restart OpenShift*. But if you just do `dnf update openvswitch` and then go away, then your node will break.